### PR TITLE
Weather app

### PR DESCRIPTION
**Pull request by elpotatoslayer**
In this iteration, I built a Weather App based on the open weather API. It includes the following features:
Fetched weather data from the above API using fetch API
Used webpack to create modular code
Implemented Javascript using es6 syntax
Used custom styling for page layout

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+    <div id="error"></div>
+
+    <div class="content" class="row">
+      <div class="col-md-6 mx-auto text-center bg-dark text-white mt-5 p-5 rounded">
+        <h1 id="w-location"></h1>
+        <h3 id="w-description"></h3>
+        <label class="switch text-center">
+          °C / °F
+          <input id='temp-btn' type="checkbox"  data-toggle="toggle>
```
Comment (bellom) [inline dist/index.html:35]: You are using `checkbox` checkout the screenshot from your web page below;
image


You should use a toggle control to determine if the result will be shown in Celsius or Fahrenheit ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/search_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/weather-app))

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @Ralph-1  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a one issue left to work on............ :slightly_smiling_face: 

# Status: Changes Required! ♻️ 

**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Changes Required

### Hi Ralph Oburu :slightly_smiling_face: 

I was not able to see the temperature in the fahrenheit unit. Currently, it only shows in celsius. A user should be able to toggle between celsius and fahrenheit.

#### That's it, make the change then submit another review form.

### Happy coding :man_technologist: 

#### ``` Uchechukwu Henry Inyama ```

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
# Changes Required :recycle:
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hello, you have been doing a tremendous job so far :ok_hand:
## To Highlight 🎉
- [ ] README is professional and includes all the info to run the app locally.
## To Improve 🙏
- [x] Please use proper Git flow. The default branch should be the development, not the feature branch.
- [x] App shows wrong weather data. It shows both Celsius and Fahrenheit in hundreds. Moreover, when I search for a city, it does not show celsius or fortnight data. Please see screenshots:
data
celsius
- [x] App should change its background based on the received data. Please see this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/appearance_requirement.png).

# Optional

- [ ] App may fetch the current geolocation of the user when it is first accessed and fetches the weather from that location

### Happy Coding ☺️

## If you have any questions you can comment here in this PR or can always connect with me on slack ( @Sajjad Ahmad ).

**Comment by mikenath223** [comment]
### STATUS: INVALID PR ❌

This PR has been marked as invalid.

Reason: 
- Please ensure all issues raised in the last review are fixed and you make a new commit after that review.

Feel free to contact me on slack(@michgolden) if you need further clarifications.

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Changes Required :red_circle:
To Highlight: :angel:

- Good job by adding toggle control :+1: 
-  Background changes based on the received data. :100: 

To improve :factory_worker: [REQUIRED]:

- [ ] App shows wrong weather data. It shows both Celsius and Fahrenheit in hundreds.


Nice to have [OPTIONAL] :

- [ ] App may fetch the current geolocation of the user when it is first accessed and fetches the weather from that location
- [ ] it has a ‘loading’ component that displays from the time the form is submitted until the information comes back from the API ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/loading_component_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/weather-app))

That's it, you can submit another code review request after making these changes :recycle: :innocent:
With all my respect :pray:
Rindra Josia

**Review by Georjane — APPROVED** [review]
## Status: PR Approved 💯 🥇 
### Hi @Ralph-1 🙌🏽 
Nice work! You have come a long way and you made all changes requested by the previous code reviewers.
The toggle button works now, although you have to click on it twice before it works. Also, the temperature conversion is now correct.
It's a thumbs up for me! 👍🏽 You can go ahead and merge this pull request.

_Good luck with your next project and Happy Coding!_ 💻 

Code Reviewer
**_Witah Georjane_**

**Pull request merged**